config/config.go

```diff
@@ -108,6 +108,11 @@ func loadForFileFromBytes(confBytes []byte, filePath string, kmsEncryptionContex
 			break
 		}
 	}
+
+	if rule == nil {
+		return nil, fmt.Errorf("error loading config: no creation rules found")
```

Comment:
Should this perhaps be `"error loading config: no matching creation rules found"`?